vmm/src/igvm/igvm_loader.rs

```diff
@@ -9,17 +9,17 @@ use crate::igvm::loader::Loader;
 use crate::igvm::IgvmLoadedInfo;
 use crate::igvm::{BootPageAcceptance, StartupMemoryType, HV_PAGE_SIZE};
 use crate::memory_manager::MemoryManager;
+use igvm::IgvmDirectiveHeader;
+use igvm::IgvmFile;
+use igvm::IgvmPlatformHeader;
+use igvm::IsolationType;
 use igvm_defs::IgvmPageDataType;
 use igvm_defs::IgvmPlatformType;
-use igvm_parser::IgvmDirectiveHeader;
-use igvm_parser::IgvmFile;
-use igvm_parser::IgvmPlatformHeader;
-use igvm_parser::IsolationType;
 
```

Comment:
This file has extra lines of whitespace which means rustfmt only sorts within the whitespace group. Can you fix that please?